api/src/controllers/board.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response, NextFunction } from 'express';
+import boardDataController from '../data-controllers/board';
+
+const getBoards = async (req: Request, res: Response, next: NextFunction) => {
+	const { userId } = res.locals.auth;
+
+	if (!userId) return res.status(400).json({ message: 'userId was null' });
+
+	const boards = await boardDataController.getUserBoards(userId).catch(next);
+
+	res.json({
+		boards,
+	});
+};
+
+const getBoard = async (req: Request, res: Response, next: NextFunction) => {
+	const { userId } = res.locals.auth;
+	const { boardId } = req.params;
+
+	if (!userId) return res.status(400).json({ message: 'userId was null' });
```

Comment:
It’s fine for now, but for future, this might be generalized into the schemas for checking the request payload and errors handling according to certain issues with the payload. 
please take a look to LIVR spec, it should cover up this topic